mtz-wizard.html

```diff
@@ -149,10 +149,10 @@
        * @param {HTMLElement[]} [steps = this.steps]
        * @return {Boolean}
        */
-      validate(steps = this.steps) {
-        return (steps || []).reduce((prev, step) => {
-          return prev && (step.validate ? step.validate() : true);
-        }, true);
+      async validate(steps = this.steps) {
```

Comment:
I'm not sure about using `async`/`await` in our open source. I would be okay with this change if it was using promises.

Reply:
I agree, this adds a dependency on es7 instead of es6 which might be more than our users are willing to bite off.